test/contrib/homeplugav.uts

```diff
@@ -48,29 +54,25 @@ HomePlugAV()/SnifferIndicate()
 pkt = HomePlugAV()/SetEncryptionKeyRequest()
 pkt.NMK = "A" * 16
 pkt.DAK = "B" * 16
-r = raw(pkt)
-r
-assert r == b"\x00P\xa0\x00\xb0R\x00AAAAAAAAAAAAAAAA\x00\xff\xff\xff\xff\xff\xffBBBBBBBBBBBBBBBB"
```

Comment:
This looks pretty unnecessary

Reply:
Made some modification, but not sure if this actually what you've requested.

Thanks in advance!